recipes/epanet-dev/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "epanet-dev" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/OpenWaterAnalytics/epanet-dev.git
+  git_tag: master
+  patches:
+    - macos.patch  # [osx]
+    - cmake.patch  # [unix]
+
+build:
+  skip: True  # [win and py27 or win and py34]
+  number: 0
+  features:
+    - vc14  # [win and py35]
+  detect_binary_files_with_prefix: True
+
+requirements:
+  build:
+    - python
+    - cmake >=3.4
+
+test:
+  commands:
+    - run-epanet3
+  requires:
+    - git
```

Comment:
Please add `- python {{ environ['PY_VER'] + '*' }}  # [win]`. This should fix the Windows issue.